core/templates/pages/exploration-player-page/layout-directives/progress-nav.component.spec.ts

```diff
@@ -254,41 +254,6 @@ describe('Progress nav component', () => {
     expect(componentInstance.skipQuestion.emit).toHaveBeenCalled();
   });
 
-  it(
```

Comment:
Why was this deleted?

Reply:
Initially, the `skipButtonIsShown` variable is set in this component, and based on that this test block was written. But I have updated it to send this value from the top hierarchical component. So this test block is not needed anymore.
Thank you